ebusd-2.1.x/de/vaillant/15.350.csv

```diff
@@ -44,16 +45,19 @@ r;wi,,OffsetDesTemp,,,,,"2300",,,temp,,,
 r;wi,,StateOfRoomCon,,,,,"2400",,,UCH,,,
 r;wi,,LegioProtectionState,,,,,"2500",,,onoff,,,
 r;wi,,StatusDcf,,,,,"2600",,,dcfstate,,,
-r;wi,,DcfDaten,,,,,"2700",,,UIN,,,
+[SW<114][HW<7102]r;wi,,DcfDaten,,,,,"2700",,,UIN,,,
+[SW=114][HW=7102]r;wi,,DesiredTempDay,,,,,"2700",,,D1C,,,
```

Comment:
usually features do not disappear with newer versions, so use `[SW>=144]` instead here and also below